libs/common/src/lib/deep-freeze.spec.ts

```diff
@@ -62,7 +62,9 @@ describe('deepFreeze', () => {
             str: 'immutable',
             num: 42,
             bool: true,
-            fn: function () {},
+            fn: function () {
```

Comment:
minor: maybe use fat arrow fn?

fn: () => {}

Reply:
The arrow function is not an issue but with your suggestion, linting will still error on `@typescript-eslint/ no-empty-function` so I will keep the explicit `return`